templates/template.html

```diff
@@ -0,0 +1,175 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+
+<head>
```

Comment:
Should we make this Mech Rock branded?

Reply:
it means, should the template.html have the Mechanical Rock logo and the look and feel (ie. the colours / font)